packages/cli/src/utils/get-env-proxy.ts

```diff
@@ -1,6 +1,14 @@
 import process from "node:process";
 
 export function getEnvProxy(): string | undefined {
-	const { HTTP_PROXY, http_proxy } = process.env;
-	return HTTP_PROXY || http_proxy;
+	const { env } = process;
+	return (
+		env.HTTP_PROXY ||
+		env.http_proxy ||
+		env.HTTPS_PROXY ||
+		env.https_proxy ||
+		env.npm_config_http_proxy ||
```

Comment:
```suggestion
```
This doesn't seem to be a valid npm config option: https://docs.npmjs.com/cli/v10/using-npm/config#proxy

the other two are fine though